base/nullable.jl

```diff
@@ -53,20 +53,20 @@ otherwise, returns `y` if provided, or throws a `NullException` if not.
 """
 @inline function get{S,T}(x::Nullable{S}, y::T)
     if isbits(S)
-        ifelse(x.isnull, y, x.value)
+        ifelse(isnull(x), y, x.value)
```

Comment:
I've opened a PR to add benchmarks: https://github.com/JuliaCI/BaseBenchmarks.jl/pull/24. Please suggest cases to add measurements for.